showcase/app/components/shw/outliner/index.gts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+import Component from '@glimmer/component';
+
+export interface ShwOutlinerSignature {
+  Blocks: {
```

Comment:
just FYI: not having even empty `Args` block here might break types in the context of an other `.gts` component, but it's fine for now